### ISSUE TITLE
OnMessageReceived instand Ratelimit [User Token]

### ISSUE THREAD

**Issue by Spielepapagei** [not-ilinked/Anarchy#3368]
I tried to log only my direct messages.
However, when using the OnMessageReceived event, it logs about five messages before disconnecting.
It immediately disconnects again when I try to restart it.

Is there a method to receive only direct messages or to avoid getting a RaidLimit?

```c#
private void MessageReceived(DiscordSocketClient client, MessageEventArgs message)
{
     Logger.Info($"Message Received from {message.Message.Author} he/she sayid {message.Message.Content}");
}
```

**Comment by verticalsync** [comment]
Most likely your IP being from a datacenter or such (e.g. a server or vpn) discord has been testing some stuff out

**Comment by Hi-ImKyle** [comment]
> Most likely your IP being from a datacenter or such (e.g. a server or vpn)

Even on a normal residential connection it still does it for me. Sometimes I never get any messages, sometimes I only get a few. It always stops after a short time though.

**Comment by Cuarento** [comment]
Do you find a solution for it? I have the same problem

**Comment by Hi-ImKyle** [comment]
I don't think this library is being updated anymore..

**Comment by verticalsync** [comment]
> I don't think this library is being updated anymore..

It hasn't been updating for a long while before you even made this issue, it's abandoned and would be too much work to make it work properly and implement new featuees.

**Comment by Hi-ImKyle** [comment]
I didn't make the issue, but your original comment about it being potentially related to the used IP belonging to a VPN/datacentre is wrong. If you say its abandoned, you may as well archive the repo..

**Comment by verticalsync** [comment]
> I didn't make the issue, but your original comment about it being potentially related to the used IP belonging to a VPN/datacentre is wrong. If you say its abandoned, you may as well archive the repo..

Yeah as I said "discord has been testing some stuff out", which was true at the time considering mine and iLinked's servers got blocked or something like that if I remember correctly, nor can I archive this repo since I don't own it.
